docs/gettinginvolved/infrastructure/vagrant.rst

```diff
@@ -47,18 +47,30 @@ Add a Vagrant Box to Atlas
 
 To add to Atlas, you will need a public link to the relevant Vagrant box.
 
-#. Visit `Atlas (flocker-dev) <https://atlas.hashicorp.com/clusterhq/boxes/flocker-dev>`_ or `Atlas (flocker-tutorial) <https://atlas.hashicorp.com/clusterhq/boxes/flocker-tutorial>`_ as applicable.
+#. Visit `Atlas <https://atlas.hashicorp.com/>`_ and sign in.
 
-#. In the dropdown of versions select "Create a version".
+#. At the top of the page, select "Development", and from the "Boxes" menu on the left, select "``clusterhq/flocker-dev``" or "``clusterhq/flocker-tutorial``" as applicable.
+
+#. In the menu at the left select "Create new version".
 
 #. Set the "Version" to the relevant version.
+
+   If releasing the development machine, ensure the version number increases using the semantic versioning format. This usually involves
```

Comment:
releasing the development machine > adding the `flocker-dev` box

New sentence on new line.